### ISSUE TITLE
Rename multiple columns at once

### ISSUE THREAD

**Issue by dgorissen** [OpenRefine/OpenRefine#901]
I have a simple use case where I just want to have all columns in lower case. With 50 odd columns doing that manually in openrefine is just painful.  

Would be nice if there was a simple way to act on multiple columns at once.

**Comment by MohitKumar1991** [comment]
Is there a way to do this now ? Looks like it didn't happen.

**Comment by wetneb** [comment]
Indeed, we still don't have a way to do that. It is not really clear to me how it would look like actually! A GREL transform that would work on column names, perhaps? Feel free to add details about your own use case for it, to motivate contributors to pick up this issue.

**Comment by connorshea** [comment]
I think a simple dialog box with a list of all the columns as editable text fields - and then a save button - would work. Right now, you have to edit each column one-at-a-time, there's no way to edit them all in a single save action.

**Comment by EricPerser** [comment]
As part of data cleaning task, I often come across fixing multiple column headers. It would be great if this feature can be added with priority (given that this request has been waiting since 2021). I appreciate those who brought this up and those who work on this feature.

**Comment by ostephens** [comment]
@EricPerser thanks for bumping this issue. For you is this just a matter of re-typing column headers, or do you want something more like a transformation as described by @dgorissen originally like "change all existing column headers to lowercase"?

**Comment by EricPerser** [comment]
Thanks for your prompt reply @ostephens ! Since I'm mainly wrangling large biological metadata, I would like to have the best of both worlds as column headers represent different health variables but mass editing goes on first priority, as you and others have pointed out.

**Comment by jfaurelacroix** [comment]
hey, funny story, that's actually a problem that I have as well! In my case, I need to be able to read two rows as the header but they are separated by a couple of rows. This means that the option to read X number of rows as the header isn't an appropriate solution in its current state.
